annotationengine/__init__.py

```diff
@@ -24,6 +32,11 @@ def create_app(test_config=None):
     app.register_blueprint(voxel.bp)
     app.register_blueprint(schemas.bp)
     app.register_blueprint(dataset_mod.bp)
```

Comment:
might not be in this PR but we should put some sort of stupid index page at the root.